notes/stv/20211231-notebook-share.txt

```diff
@@ -0,0 +1,261 @@
++--------------------------------------+-----------------------------+-------+-------------+-----------+-----------+------------------+------+-------------------+
+| 2e46b5a5-c5d9-44c0-b11c-310c222f4818 | aglais-data-gaia-dr2-6514   |   512 | CEPHFS      | available | True      | cephfsnativetype |      | nova              |
+| ca8231c3-1f5c-4ebf-8ec0-d3cfe2629976 | aglais-data-gaia-edr3-11932 |   540 | CEPHFS      | available | True      | cephfsnativetype |      | nova              |
+| d583565e-de86-46df-9969-f587e4d61a37 | aglais-data-gaia-edr3-2048  |  1024 | CEPHFS      | available | True      | cephfsnativetype |      | nova              |
+| 9d745a5b-7d98-421c-a16e-d1ac9fdeebc8 | aglais-data-gaia-edr3-4096  |  1024 | CEPHFS      | available | True      | cephfsnativetype |      | nova              |
+| 2e877d53-40b9-47e6-ae20-b6d3e1b9a9ae | aglais-data-gaia-edr3-8192  |  1024 | CEPHFS      | available | True      | cephfsnativetype |      | nova              |
+| ba66d6db-7d85-44c4-bb95-7410a000f6b7 | aglais-data-panstarrs-ps1   |   300 | CEPHFS      | available | True      | cephfsnativetype |      | nova              |
+| e65c0e26-957f-4ab0-94af-bb36b5a63285 | aglais-data-testing         |    10 | CEPHFS      | available | True      | cephfsnativetype |      | nova              |
+| 9dc3016a-f010-48bc-89fc-a9cbd688b7cc | aglais-data-twomass-allsky  |    40 | CEPHFS      | available | True      | cephfsnativetype |      | nova              |
+| 8f0b3452-3c66-4e65-8815-15eb73988b3e | aglais-data-wise-allwise    |   350 | CEPHFS      | available | True      | cephfsnativetype |      | nova              |
+| 17310ca5-c9a6-43bb-987b-de543d453535 | aglais-test-data            |  1024 | CEPHFS      | available | True      | cephfsnativetype |      | nova              |
+| eeb95821-f8f5-40d0-a04f-ea9cbf6e538b | aglais-tools                |  1024 | CEPHFS      | available | True      | cephfsnativetype |      | nova              |
+| 7b03dcf9-6806-44a0-b87f-56528b50338f | aglais-user-dcr             |  1024 | CEPHFS      | available | True      | cephfsnativetype |      | nova              |
+| 6852b819-7395-4786-80c0-06fa9cebcc65 | aglais-user-nch             | 10240 | CEPHFS      | available | True      | cephfsnativetype |      | nova              |
+| fe63568a-d90c-4fb0-8979-07504328809d | aglais-user-stv             |  1024 | CEPHFS      | available | True      | cephfsnativetype |      | nova              |
+| ff351afd-1f06-4d02-9f53-cbe20b0676cc | aglais-user-zrq             |  1024 | CEPHFS      | available | True      | cephfsnativetype |      | nova              |
++--------------------------------------+-----------------------------+-------+-------------+-----------+-----------+------------------+------+-------------------+
+
+
+
+
+# -----------------------------------------------------
+# Create a new 4Tbyte share for notebooks.
```

Comment:
4Tbyte is a _**lot**_ of space just for  notebooks.
Is this because we are expecting the notebooks to store the pickled results and plots as well ?

Reply:
To be honest I just picked an arbitrary size that would be more than enough to ensure we don't run out of space. The notebooks do seem to store the output in the same file which makes them grow a bit, however I probably agree, I doubt we'd get anywhere close to 4GB. Happy to reduce that to a lower value, although I'm not sure exactly would it would be. We'd probably have to run all notebooks and get the cumulative size, and add to that a buffer for what we'd expect from multiple users over time.

Reply:
It is fine for now, just a bit unexpected.
Let's see how much space this ends up using, and if we need to we can grow or shrink the size to match.
I'm in the middle of transferring the shares to the new cloud, so hopefully we will have better bandwidth to the shares.

Reply:
This isn't a show-stopper. I've approved the PR so go ahead and merge it when you are ready.